cli/wallet/wallet.go

```diff
@@ -103,12 +107,16 @@ func NewCommands() []*cli.Command {
 		txctx.AwaitFlag,
 		inFlag,
 		&flags.AddressFlag{
-			Name:    "address",
-			Aliases: []string{"a"},
-			Usage:   "Address to use",
+			Name:     "address",
+			Aliases:  []string{"a"},
+			Required: true,
+			Usage:    "Address to use",
 		},
 	}
-	signFlags = append(signFlags, options.RPC...)
+	rpcFlagOriginal, _ := options.RPC[0].(*cli.StringFlag)
+	rpcFlag := *rpcFlagOriginal
+	rpcFlag.Required = false
+	signFlags = append(signFlags, &rpcFlag, options.RPC[1])
```

Comment:
Could you please check that there's a test for this command that omits RPC flag? This test should pass.